Tests/Cell.py

```diff
@@ -279,7 +277,7 @@ def test_relative_proportional_spread_happy_case(self):
             Member.of(DIMENSION_RPS2_NAME, "e3")])).to_mdx()
 
         values = self.tm1.cubes.cells.execute_mdx_values(mdx)
-        
+
```

Comment:
@scrambldchannel
nevermind these formatting changes. I didn't actually change that stuff.
PyCharm formats the code pep-8 compliant when you press CTRL + ALT + L.

Reply:
No problem! I was going to ask about using code formatters actually. I'll run autopep8 on my code going forward :)